build.gradle

```diff
@@ -459,6 +461,53 @@ bootWithCCD {
     }
 }
 
+cftlibTest {
+  // IDAM simulator will be started on port 5000,
+  // S2S simulator on port 8489
+  authMode = uk.gov.hmcts.rse.AuthMode.Local
```

Comment:
To view jurisdictions on XUI, Try this in the cftlibTest task.
```groovy
environment 'XUI_JURISDICTIONS', 'ST_CIC,ST_SEND,ST_PHL,ST_CS,ST_MH'
```

Similar to what's in the [bootWithCCD](https://github.com/hmcts/sptribs-case-api/blob/c0b384c6d5aa3aa5cf9879700660a6e65b106abc/build.gradle#L447) task.